hardcodes.R

```diff
@@ -365,22 +355,23 @@ inc_ncb_hi_required <- unnest_longer(inc_ncb_hi_required_prep, ProjectType) %>%
 
 sys_comp_filter_choices1 = c(
   "Age", 
-  "Domestic Violence Status",
+  "Domestic Violence",
   "Gender",
-   #"Homelessness Type", # Victoria, 8/15/24: Not including this for Launch
+  "Homelessness Type",
```

Comment:
are we including this?

Reply:
No I think this got overwritten accidentally. I'll fix it